recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
     - 0001-fix-arrowfs-skip-arrow-fs-test-on-windows.patch
 
 build:
```

Comment:
We need to add a skip for the older arrow pins as they don't support Python 3.12 (due to depending on an old pandas pin)

Reply:
Can you please elaborate what exactly needs to be added

Reply:
Probably something along the lines of
```
skip: true  # [py==312 and libarrow in ['10',..]
```

I would test that locally in 1h and report back if you the patience.

Reply:
Sure, I'm in no hurry

Reply:
Looking more into the duckdb sources, we can massively change this recipe.

Reply:
Would be nice to have a non-Python build and the CLI.